zerver/openapi/zulip.yaml

```diff
@@ -19231,6 +19425,61 @@ components:
 
             New in Zulip 8.0 (feature level 191). Previously, groups
             could be mentioned if and only if they were not system groups.
+    Invite:
+      type: object
+      description: |
+        A dictionary containing details about an invitation.
+      additionalProperties: false
+      properties:
+        id:
+          type: integer
+          description: |
+            The unique ID of the invitation link object.
+        invited_by_user_id:
+          type: integer
+          description: |
+            The [user ID](/api/get-user) of the user who created the invitation.
+
+            **Changes**: New in Zulip 3.0 (feature level 22). Previously, `ref`
+            field was used which contained the Zulip display email address of the
+            user who created the invitation.
+        invited:
+          type: integer
+          description: |
+            The UNIX timestamp for when the invitation was created, in UTC seconds.
+        expiry_date:
+          type: integer
+          nullable: true
+          description: |
+            The UNIX timestamp for when the invitation will expire, in UTC seconds.
+            The value will be `null` if the invitation is never-expiring.
+        invited_as:
+          type: integer
+          enum:
+            - 100
+            - 200
+            - 300
+            - 400
+            - 600
+          description: |
+            The [organization-level role](/api/roles-and-permissions) that the invitee
+            will be assigned to after accepting the invitation.
+        email:
+          type: string
+          description: |
+            The email address of the invitee to which the invitation is sent. This will
+            not be present when `is_multiuse` is `true` (i.e. invitation is generated as a
+            reusable link).
```

Comment:
-> "The email address the invitation was sent to. This will not be present when `is_multiuse` is `true` (i.e. the invitation is a reusable invitation link)."